BrainPortal/app/models/tool_config.rb

```diff
@@ -465,4 +465,89 @@ def to_carmin #:nodoc:
     }
   end
 
+  ##################################################################
```

Comment:
Some missing `#:nodoc:` in this file.

Reply:
Yes, I will do a full review of all the #:nodoc: of all new methods soon (some of them need to actually be documented)